pkgs/os-specific/linux/amdgpu-pro/default.nix

```diff
@@ -172,15 +172,15 @@ in stdenv.mkDerivation rec {
     # this could also be done with LIBGL_DRIVERS_PATH, but it would need to be
     # set in the user session and for Xorg
     then ''
-      expr1='s:/opt/amdgpu/lib/x86_64-linux-gnu/dri\0:/run/opengl-driver/lib/dri\0\0\0\0\0\0\0\0\0\0\0:g'
-      expr2='s:/usr/lib/x86_64-linux-gnu/dri[\0\:]:/run/opengl-driver/lib/dri\0\0\0\0:g'
+      expr1='s:/opt/amdgpu/lib/x86_64-linux-gnu/dri\0:/run/hardware-drivers/lib/dri\0\0\0\0\0\0\0\0\0\0\0:g'
```

Comment:
I'm pretty sure these changes will clobber the binary, the original code contains padding to match the length of the string and the new path is longer

Reply:
The legacy link will be retained. So it might be an exception